cd /t3home/fglessge/CMSSW_10_1_0/src/Xbb/python/
Task: prep
Pwd: /t3home/fglessge/CMSSW_10_1_0/src/Xbb/python
Host: t3wn36
User: fglessge
StartTime: 2022-02-11 17:53:35

Parsing files in Zll2016config...

python ./prepare_environment_with_config.py --fileList base64:eNq1zsEKgkAQBuAnsn/GtTA6RuCl8GAevAwuDK4Q7rK7Sr595+jcE3wfUvZRMUW/BgS3J3HzNCU4a7HkNbw0oW+sbX3Kj3HxKIlP6JllMxgG6dZFr89b19Z3Fjad9kXYs5vHGsXG4DMdSyMlsTEViIiQo6rwIXqfL/ifLvrO/F2omH8KH/wAUwk= --sampleIdentifier ZZ_TuneCUETP8M1_13TeV-pythia8 --config /t3home/fglessge/CMSSW_10_1_0/src//Xbb/python/logs_Zll2016//prep-v1/config//submission_20220211_123252.ini
Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3011] Unable to open file /eos/cms/store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2016/V11_v3/ZZ_TuneCUETP8M1_13TeV-pythia8/-v1/190523_201334/0000/tree_1.root; No such file or directory

Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3011] Unable to open file /eos/cms/store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2016/V11_v3/ZZ_TuneCUETP8M1_13TeV-pythia8/_ext1-v1/190523_201411/0000/tree_1.root; No such file or directory

start prepare_environment_with_config.py
start copytreePSI.py
len(filenames) 2 /store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2016/V11_v3/ZZ_TuneCUETP8M1_13TeV-pythia8/-v1/190523_201334/0000/tree_1.root 1
##### COPY TREE - BEGIN ######
remove_branches: [''] len(remove_branches): 1
input file NOT EXISTING: root://eoscms.cern.ch//store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2016/V11_v3/ZZ_TuneCUETP8M1_13TeV-pythia8/-v1/190523_201334/0000/tree_1.root
input file NOT EXISTING: root://eoscms.cern.ch//store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2016/V11_v3/ZZ_TuneCUETP8M1_13TeV-pythia8/_ext1-v1/190523_201411/0000/tree_1.root
##### COPY TREE - END ######
end prepare_environment_with_config.py
--------------------------------------------------------------------------------
exit code: 0
EndTime: 2022-02-11 17:53:56
duration (real time): 0 minutes
--- OK ---

Exiting runAll.sh

